dev-tools/omdb/src/bin/omdb/nexus.rs

```diff
@@ -103,18 +104,55 @@ enum BlueprintsCommands {
     Import(BlueprintImportArgs),
 }
 
-#[derive(Debug, Args)]
+#[derive(Debug, Clone, Copy)]
+enum BlueprintIdOrCurrentTarget {
+    CurrentTarget,
+    BlueprintId(Uuid),
+}
+
+impl FromStr for BlueprintIdOrCurrentTarget {
+    type Err = uuid::Error;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        if s == "current-target" {
```

Comment:
Ha -- I was tempted to suggest `current`!

Reply:
That also came to mind, but I figured "target" is more descriptive about the fact that blueprints are a means to express desired state? But you're the blueprint expert lol

Reply:
Hah, I have gone back and forth on this since yesterday. When I'm leaning toward target, it's for exactly the reason you mention (it's more descriptive that a blueprint is aspirational and may not match the system), and when I'm leaning toward current, it's because that word seems more natural if you're not steeped in the blueprint system ("What is the current blueprint?"). I think I'm coming down on the side of `target`; I'll go ahead and make that change on this PR, but obviously it's really easy to switch if there's a strong argument for `current` (or something else entirely) instead.

Reply:
I'll throw out a "something else entirely" - we could accept either.

Reply:
Both is fine by me, it's not as if there's any potential for confusion.

Reply:
:+1: I'll go with that (added all three in [8bbd581](https://github.com/oxidecomputer/omicron/pull/5543/commits/8bbd581a3b477b3f30e96e9cc344d9246f293de8)).